eclipse-theia/theia
# Add Neo 2 Layout data

This is for https://github.com/gitpod-io/gitpod/issues/1363 where I have executed the generate-layout script on an Ubuntu 20.10. (Windows 10 does not support Neo 2 layouts afaik)

Since I just followed instructions and uploaded a generated file, I don't think the Eclipse Contributor Agreement (ECA) is required. If you think otherwise I can of course do that step.

#### What it does
Adds the result of

```yarn --cwd packages/core generate-layout --output src/common/keyboard/layouts/de-Neo_2-pc.json```

#### How to test

Choose Neo 2 Keyboard in (linux) keyboard settings. Then check if shortcuts work. (e.g. Copy + Paste) This is of course hard to do if you don't have the layout memorized :-/

#### Review checklist


No. I have just uploaded the generated file.

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
